scipy/interpolate/tests/test_interpolate.py

```diff
@@ -2581,6 +2582,32 @@ def test_fillvalue_type(self):
         RegularGridInterpolator(points, values)
         RegularGridInterpolator(points, values, fill_value=0.)
 
+    def test_broadcastable_input(self):
+        # input data
+        np.random.seed(0)
+        x = np.random.random(10)
+        y = np.random.random(10)
+        z = np.hypot(x, y)
+
+        # x-y grid for interpolation
+        X = np.linspace(min(x), max(x))
+        Y = np.linspace(min(y), max(y))
+        X, Y = np.meshgrid(X, Y)
+        XY = np.vstack((X.ravel(), Y.ravel())).T
+
+        for itype in (NearestNDInterpolator, LinearNDInterpolator,
+                      CloughTocher2DInterpolator):
+            interp = itype(list(zip(x, y)), z)
+            # single array input
+            interp_points0 = interp(XY)
+            # broadcastable input
+            interp_points1 = interp((X, Y))
```

Comment:
This is a tuple of 1-D array input. I think what should be tested here is two positional args, both 1-D arrays. So `interp(X, Y)`. And same for the line below.

Reply:
Oh, OK. I fixed it.